pkg/cli/admin/nodeimage/mirrorbuilder.go

```diff
@@ -0,0 +1,64 @@
+package nodeimage
+
+import (
+	ocpv1 "github.com/openshift/api/config/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+)
+
+func getIdmsContents(idmsMirrorSetList *ocpv1.ImageDigestMirrorSetList, icspMirrorSetList *ocpv1.ImageContentPolicyList) ([]byte, error) {
+	imageDigestSet := ocpv1.ImageDigestMirrorSet{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: ocpv1.GroupVersion.String(),
+			Kind:       "ImageDigestMirrorSet",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "image-digest",
+			// not namespaced
+		},
+	}
+
+	// Add mirrors from IDMS
+	if idmsMirrorSetList != nil {
+		for _, idms := range idmsMirrorSetList.Items {
+			for _, digestMirror := range idms.Spec.ImageDigestMirrors {
+				imageDigestSet.Spec.ImageDigestMirrors = append(imageDigestSet.Spec.ImageDigestMirrors, digestMirror)
+			}
+		}
+	}
+
+	// Add mirrors from ICSP
+	if icspMirrorSetList != nil {
```

Comment:
I know this has been already previously discussed - and I keep finding a little bit awkward chaining both the sources. Since ICSP is deprecated,I think that at least logging its usage could make it more evident